### ISSUE TITLE
first pre-trained using only frame prediction loss

### ISSUE THREAD

**Issue by wenxiaopeng-del** [ktr-hubrt/MPN#20]
Before meta-training, the AE backbone is first pre-trained using only frame prediction loss(Eq. 4)." What are the exact details, and approximately how many epochs do you need to train.
<img width="376" alt="屏幕截图 2021-08-17 163244" src="https://user-images.githubusercontent.com/61273910/129692308-d1c6e304-8ed2-4c47-b556-c05ecfc95037.png">

**Comment by ktr-hubrt** [comment]
I train the base model until the loss convergence. Cause i modify the dataloader, the initial training epoch is not reliable.
